CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Input props `large` and `xLarge` are now `size="large"` and `size="x-large"``.
+- Input's label has now a condition for displaying.
```

Comment:
O que acha de:

```
### Fixed
- Input: if the prop `label` is not passed, no HTML for label is rendered.

### Changed
```

Colocar em outra sessão como pede o https://keepachangelog.com/en/1.0.0/

![image](https://user-images.githubusercontent.com/284515/38212911-df518a7a-3695-11e8-9733-3ff01dce811b.png)

Reply:
### Changed
- `Added` Input: if the prop `label` is not passed, no HTML for label is rendered.
- `Removed` Dropdown and Input prop `block`. They are always block.
- `Removed` Dropdown and Input props `short` and `long`. Their widths should be defined by their parents.
- `Changed` Input props `large` and `xLarge` are now `size="large"` and `size="x-large"``.

Reply:
Boa, o tipo da change é um nome de uma sessão:

```
### Changed
- Input: if the prop label is not passed, no HTML for label is rendered.
- Input props large and xLarge are now size="large" and size="x-large".

### Removed
- Dropdown and Input prop block. They are always block.
- Dropdown and Input props short and long. Their widths should be defined by their parents.
```

Reply:
Boa. Atualizando.